frontend/dispatcher.lua

```diff
@@ -619,40 +620,49 @@ function Dispatcher:execute(ui, settings, gesture)
         if settingsList[k] ~= nil and (settingsList[k].conditions == nil or settingsList[k].conditions == true) then
             -- Be sure we don't send a document setting event if there's not yet or no longer a document
             if ui.document or (not settingsList[k].paging and not settingsList[k].rolling) then
+                Notification:setNotifySource(Notification.SOURCE_DISPATCHER)
+                if settingsList[k].notify_func then
+                    Notification:notify(settingsList[k].notify_func())
```

Comment:
We/you had gotten rid of all notify_func. You resurrected 2 of them.

Reply:
Sorry, I did not commit this change (was a side effect of rebase).

Reply:
I seem to see a bit too much diff from what's expected with dispatcher.lua :/ ?
Can you git checkout master, git pull (so you update your master to include the dispatcher.lua changes from the other PR), git checkout this branch and git rebase master ?

Reply:
I did this and then a git push origin bottomConsistentNotificacion. All I get is `Everything is up-to-date`.
Should I do a `git push -f origin ...`?

Reply:
But is your master actually up-to-date? (e.g., `git fetch upstream && git reset --hard upstream/master`).

Sidebar: I actually just do that, without going through my local master branch.

`git fetch upstream && git rebase upstream/master`

Reply:
Well, looks allright, you have the  `Dispatcher: fix possible crash when no document (koreader#7716)` in your commit history of this branch and of your master branch.
Do you also see the added noise locally? git diff master frontend/dispatcher.lua ?

If yes, not sure... May be re-apply your few lines change on a clean dispatcher.lua:
```
git checkout master frontend/dispatcher.lua
   < re-add your changes >
git add frontend/dispatcher.lua
git commit
```

Reply:
my `git diff master frontend/dispatcher.lua` shows exactly two lines more and all the lines shifted right.

Reply:
@niluje: I did a git checkout master;  git fetch upstream ; git merge upstream/master before.